### PR TITLE
Adds spanish translations

### PR DESCRIPTION
**Pull request by Enchufadoo**
### Changes
Spanish translations

### Testing
None

### Others
- Fixes #0
- Connects #108

### CODE REVIEW

File: app/src/main/res/value-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  <string name="volume">Volumen de sonido</string>
+  <string name="play_pause">Reproducir o pausar</string>
+  <string name="open_drawer">Abrir el menú de navegación</string>
+  <string name="close_drawer">Cerrar el menú de navegación</string>
+  <string name="notification_channel_default__name">Reproducción de medios</string>
+  <string name="notification_channel_default__description">Controles de reproducción</string>
+  <string name="pause">Pausar</string>
+  <string name="play">Reproducir</string>
+  <string name="stop">Detener</string>
+  <string name="birds">Pajaros</string>
```
Comment (ashutoshgngwr) [inline app/src/main/res/value-es/strings.xml]: Pájaros

Comment (ashutoshgngwr) [inline app/src/main/res/value-es/strings.xml]: Pájaros

File: app/src/main/res/value-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  <string name="preset_name_cannot_be_empty">El nombre del preajuste no puede estar vacío</string>
+  <string name="preset_saved">¡Preajuste guardado!</string>
+  <string name="saved_presets">Preajustes Guardados</string>
+  <string name="delete_preset">Borrar Preajuste</string>
+  <string name="preset_delete_confirmation">¿Confirma borrar %s?</string>
+  <string name="delete">Borrar</string>
+  <string name="preset_deleted">¡Preajuste borrado!</string>
+  <string name="nothing_saved_yet">¡Nada ha sido guardado aún!</string>
+  <string name="sleep_timer">Temporizador</string>
+  <string name="schedule">Agenda</string>
```
Comment (ashutoshgngwr) [inline app/src/main/res/value-es/strings.xml]: Agendar

**Comment by Enchufadoo** [comment]
Hi! made another merge fixing the typos and setting the values to es-ar

**Comment by ashutoshgngwr** [comment]
@Enchufadoo I have added a link to your profile in the application's about page. I hope that's not a problem. https://github.com/ashutoshgngwr/noice/commit/dbb985760a3a200ae3589f554b9dbedca8de5585

**Comment by Enchufadoo** [comment]
Great, it's not problem at all. Let me know if you need anything.

**Pull request merged**